.clang-format

```diff
@@ -1,4 +1,4 @@
-ColumnLimit: 120
+ColumnLimit: 150
```

Comment:
In more places than not we're exceeding 120, so I thought maybe it makes sense to bump it up a bit.